deluge/Dockerfile

```diff
@@ -2,31 +2,40 @@ FROM ubuntu:bionic
 MAINTAINER neggelandia@gmail.com
 
 # ARG DELUGE_URL
-ENV DELUGE_URL ${DELUGE_URL:-https://ftp.osuosl.org/pub/deluge/source/deluge-1.3.15.tar.bz2}
+ENV DELUGE_URL ${DELUGE_URL:-https://ftp.osuosl.org/pub/deluge/source/2.0/deluge-2.0.3.tar.xz}
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN \
   # install build and runtime dependencies
   apt-get -y update && \
-  apt-get -y install --no-install-recommends curl \
-    python \
-    python-twisted \
-    python-openssl \
-    python-setuptools \
-    gettext \
+  apt-get -y install python3-libtorrent \
+    python3-geoip python3-dbus \
+    python3-gi \
+    python3-gi-cairo \
+    python3-setuptools \
+    gir1.2-gtk-3.0 \
+    gir1.2-appindicator3 \
+    libnotify4 \
+    librsvg2-common xdg-utils \
+    python3-dev \
+    curl \
+    python3-pip \
+    python3-markupsafe \
     intltool \
-    python-xdg \
-    python-chardet \
-    geoip-database \ 
-    python-geoip \
-    python-libtorrent \
-    python-mako \
+    xz-utils \
+    build-essential \
+    libtiff5-dev libjpeg8-dev libopenjp2-7-dev zlib1g-dev \
+    libfreetype6-dev liblcms2-dev libwebp-dev tcl8.6-dev tk8.6-dev python3-tk \
+    libharfbuzz-dev libfribidi-dev libxcb1-dev \ 
```

Comment:
Are you sure all these GUI related dependencies are really required?

Reply:
They are required to build [Pillow](https://pillow.readthedocs.io/en/latest/installation.html)

Reply:
Okay

Reply:
I tried building your branch and I got it to build without having to build Pillow. The new documentation on how to build Deluge 2.0 seems to be quite lacking.

Reply:
I can see that you made the image quite smaller. I think we can close this PR.